### ISSUE TITLE
Transphyseal fracture of the distal humerus

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6433]
**Preferred term label:** Transphyseal fracture of the distal humerus

**Synonyms**

**Definition (free text, please give PubMed ID)** A transphyseal fracutre of the distal humerus. This typically occurs in children younger than 3 years scondary to birth trauma, nonaccidental trauma, or a fall from a small height. 

**Parent term (use hpo.jax.org/app)** Fractured humerus HP:0041055

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@JillMiller  added, thanks!
